deepchem/models/dft/dftxc.py

```diff
@@ -130,7 +144,7 @@ def __init__(self,
                  n_tasks: int = 0,
                  log_frequency: int = 0,
                  mode: str = 'classification',
-                 device: Optional[torch.device] = None,
+                 device: str = "cpu",
```

Comment:
nit: any advantage of having device as `str` rather than `torch.device`?

Reply:
There seems to be no advantage, and there is some mypy issues on this. I will resolve it soon and tag for merging